airbot_error_manager:
  error_list:
    e_error:
      e_04: # Left Motor 구속 및 제어 불가 시 에러
        sub_topic: "error/e_code/left_motor_stuck"
        error_code: "E04"
      e_04_1: # Right Motor 구속 및 제어 불가 시 에러
        sub_topic: "error/e_code/right_motor_stuck"
        error_code: "E04-1"
      e_04_2: # Left Motor 과열 시 에러
        sub_topic: "error/e_code/left_motor_overheat"
        error_code: "E04-2"
      e_04_3: # Right Motor 과열 시 에러
        sub_topic: "error/e_code/right_motor_overheat"
        error_code: "E04-3"
      e_05: # Front Lidar 이물질로 인해 기능을 상실한 경우 에러
        sub_topic: "error/e_code/scan_dirty_front"
        error_code: "E05"
      e_06: # Back Lidar 이물질로 인해 기능을 상실한 경우 에러
        sub_topic: "error/e_code/scan_dirty_back"
        error_code: "E06"
      e_07: # Docking Station을 확인하지 못하는 경우 에러
        sub_topic: "error/e_code/docking_station"
        error_code: "E07"
      e_08: # Docking Station에서 충전이 불가능한 경우 에러
        sub_topic: "error/e_code/charging"
        error_code: "E08"
    f_error:
      f_01: # (충전시) 배터리 과전류에 의한 화재 및 파손 방지 에러
        sub_topic: "error/f_code/battery_charging_overcurrent"
        error_code: "F01"
      f_01-1: # (방전시) 배터리 과전류에 의한 화재 및 파손 방지 에러
        sub_topic: "error/f_code/battery_discharging_overcurrent"
        error_code: "F01-1"
      f_07: # 1D ToF 센서와 통신이 안될 경우 에러
        sub_topic: "error/f_code/top_tof"
        error_code: "F07"
      f_09_2: # Camera 센서와 통신이 안될 경우 에러
        sub_topic: "error/f_code/camera"
        error_code: "F09_2"
      f_11: # Right 구동모터 통신 불가 에러
        sub_topic: "error/f_code/right_motor"
        error_code: "F11"
      f_12: # Left 구동모터 통신 불가 에러
        sub_topic: "error/f_code/left_motor"
        error_code: "F12"
      f_13: # Front Lidar 센서와 통신이 안될 경우 에러
        sub_topic: "error/f_code/scan_front"
        error_code: "F13"
      f_14: # Back Lidar 센서와 통신이 안될 경우 에러
        sub_topic: "error/f_code/scan_back"
        error_code: "F14"
      f_15: # 충전시 배터리 과열 에러
        sub_topic: "error/f_code/battery_charging_overheat"
        error_code: "F15"
      f_16: # 방전시 배터리 과열 에러
        sub_topic: "error/f_code/battery_discharging_overheat"
        error_code: "F16"
      f_17: # Left Multi TOF 센서와 통신이 안될 경우 에러
        sub_topic: "error/f_code/bot_tof_left"
        error_code: "F17"
      f_17_1: # Right Multi TOF 센서와 통신이 안될 경우 에러
        sub_topic: "error/f_code/bot_tof_right"
        error_code: "F17-1"
    s_error:
      s_02: # 배터리 저전력 에러
        sub_topic: "error/s_code/low_battery"
        error_code: "S02"
      s_03: # 배터리 방전 경고 에러
        sub_topic: "error/s_code/discharging_battery"
        error_code: "S03"
      s_05: # 현재 구역에서 다른 구역으로 이동이 불가한 경우 목적지 도달 불가 에러
        sub_topic: "error/s_code/unreachable_goal"
        error_code: "S05"
      s_05_2: # 사용자가 설정한 목적지가 아닌 대체 목적지에 도달하였을 경우 목적지 임시 변경 에러
        sub_topic: "error/s_code/change_temp_goal"
        error_code: "S05-2"
      s_07: # 제품의 전도 발생 시 기기 넘어짐 에러
        sub_topic: "error/s_code/fall_down"
        error_code: "S07"
      s_08: # 도킹을 못하는 경우 에러
        sub_topic: "error/s_code/unable_to_dock"
        error_code: "S08"
      s_10_2: # SoC 보드의 과열에 의한 화재 및 파손 방지 시스템 보드 과열 에러
        sub_topic: "error/s_code/board_overheat"
        error_code: "S10-2"
      s_11: # 충전부 과열에 의한 화재 및 파손 방지 에러
        sub_topic: "error/s_code/station_overheat"
        error_code: "S11"
